trace.go

```diff
@@ -138,16 +152,37 @@ func (s *Traces) Func() string {
 	return (*s)[0].Func
 }
 
-// String returns debug-friendly representaton of traces
+// Func returns just function name
+func (s *Traces) FuncName() string {
```

Comment:
No reason to use a pointer receiver here - `Traces` is already almost-pointer by design, and it gets in the way of implementation by necessitating dereference. That's true for all non-mutating methods on `Traces`